main.py

```diff
@@ -312,15 +368,16 @@ def morning_goodness():
 reply_handler = MessageHandler(Filters.reply & Filters.group, reply)
 dispatcher.add_handler(reply_handler)
 
-group_handler = MessageHandler(Filters.group, group)
+group_handler = MessageHandler(Filters.group & Filters.text, group)
```

Comment:
Why

Reply:
So we don't need that if condition to check if its None or not. Since I think it will not be None